tools/oneclickapps/README.md

```diff
@@ -0,0 +1,69 @@
+
+`.github/workflows/update-oneclick-apps.yml` is configured to invoke
+`prepare_digital_ocean_one_click_app_release.py` as a GitHub action during each Zulip
+server release.
+
+You also need to set the following secrets in your GitHub repository to make the action
+work correctly. These secrets are passed as environment variables to the GitHub action.
+
+* `DIGITALOCEAN_API_KEY` - DigitalOcean API KEY used for creating droplets, snapshots etc.
+* `ZULIP_SITE` - The URL of the Zulip realm where the message containing droplet/image details
+  should be sent.
+* `ZULIP_API_KEY` - The API key of the Zulip Bot used for sending messages.
+* `ZULIP_STREAM` - The stream to which the messages should be sent.
+* `ZULIP_EMAIL` - The email of the Zulip bot.
+
+Also pass the following as environment variables in `.github/workflows/update-oneclick-apps.yml`.
+*  `PYTHON_DIGITALOCEAN_REQUEST_TIMEOUT_SEC` -  This configures the maximum number of seconds
+  to wait before the requests made by `python-digitalocean` timeout. If not configured, it's
+  common for the requests to take 20+ minutes before getting timed out.
+
+### Verifying the one click app image
+* The action will send the image name and test droplet details to the stream configured in the
+  above steps.
+* SSH into the test droplet by following the instructions in the message.
+* After logging into the test droplet, exit the installer and run the following script.
+
+  https://raw.githubusercontent.com/digitalocean/marketplace-partners/master/scripts/img_check.sh
+
+  This script checks whether the image created is valid. It is also run by the DigitalOcean team
+  before they approve the image submission in the one click app marketplace.
+* If there are no errors (you can ignore most of the warnings), exit the SSH connection and
+  reconnect.
+* Populate the details asked by the installer and verify that the installer completes successfully.
+  If there are errors see the section below.
+* Use the link generated by the installer to create a new Zulip organization. Do some basic
+  testing like sending a bunch of messages and reloading the webpage.
+* If there are no issues, submit the image in the
+  [DigitalOcean vendor portal](https://marketplace.digitalocean.com/vendorportal). You need to be
+  added to the vendor portal team for doing this. Ask Tim to add you if required. During the submission,
+  make sure to update the blog post URL if it's a major release.
+* Keep checking the vendor portal for change in status of the submission. DigitalOcean does nominally send
+  emails when there are updates on the submission, but we have found the emails to not always arrive.
+* If there are any issues with submission, rebuild the image by manually invoking the script and
+  resubmit. The issues we have seen mostly in the past are caused by the dependencies getting outdated
+  by the time the DigitalOcean team run the checks. In that case you have to just rebuild the image
+  by invoking the script.
+
```

Comment:
Since we always reuse the `oneclicktest.zulipdev.org` domain, I think there's a step missing to delete the droplet when you're done testing?

Reply:
> Since we always reuse the oneclicktest.zulipdev.org domain, I think there's a step missing to delete the droplet when you're done testing?


Yeah. Good catch. I have added the step asking the person to delete the droplet after testing.